### DISCUSSION TITLE
Solution to Hubs not installing, failing on issuing SSL certs

### DISCUSSION THREAD

**Discussion by blairmacintyre** [mozilla/hubs#4147] [Hubs Cloud Questions] [answered]
I've been having problems with installing hubs on a couple of domains in a few AWS accounts over the past year.  No matter what I did, I would always have things fail when trying to issue SSL certs to one of the domains.

I've finally figured out the problem, it's definitely "user error by AWS novice" (me).

One of these domains had been moved from another registrar into Route 53;  one had been moved between AWS accounts.  In both cases, new hosted zones were created.  I assumed (wrongly) that AWS would take care of making sure the new NS records would be reflected in the DNS system, but that is not the case.  You need to make sure that the NS record (typically 4 AWS DNS servers) match the DNS server list in the domain registration part of Route 53.  If they don't match, edit the DNS server list in the registration part of the system.

I figured this out by accident because I was checking something in https://www.whatsmydns.net/ and realized that the NS records were wrong (they didn't match the NS records in my hosted domain list).

Once I fixed the DNS servers, and waited a few days for them to propagate across the net, both of these problematic hubs cloud instances installed without these errors.

**Comment by pattersonbl2** [comment, marked as answer]
I will make this as a solution for people who run into the same issue
